README.md

```diff
@@ -6,75 +6,169 @@
 *ksonnet* is a framework for writing, sharing, and deploying Kubernetes application manifests. With its CLI, you can generate a complete application from scratch in only a few commands, or manage a complex system at scale.
 
 Specifically, ksonnet allows you to:
-* *Reuse* common manifest patterns (within your app or from external libraries)
-* *Customize* manifests directly with powerful object concatenation syntax
-* *Deploy* app manifests to multiple environments
-* *Diff* across environments to compare two running versions of your app
-* *Track* the entire state of your app manifests in version control
+* **Reuse** common manifest patterns (within your app or from external libraries)
+* **Customize** manifests directly with powerful object concatenation syntax
+* **Deploy** app manifests to multiple environments
+* **Diff** across environments to compare two running versions of your app
+* **Track** the entire state of your app configuration in version controllable files
+
+All of this results in a more iterative process for developing manifests, one that can be supplemented by continuous integration (CI).
 
 *STATUS: Development is ongoing—this tool is pre-alpha.*
 
 ## Install
```

Comment:
Probably should make a note to add a "releases" section when we cut a release, too.

Reply:
^ this is in the *Changelog* section at the bottom of the README